ui/packages/ui/src/main/webapp/js/views/installer/SsoConfiguration.view.js

```diff
@@ -141,9 +140,7 @@ define([
         return
       }
 
-      if (this.modified) {
-        this.persistConfig()
-      }
+      this.persistConfig()
```

Comment:
📘 Always save config so the resulting metatype configurations have a configuration to use.